### PR TITLE
Add cheerio for views tests

### PR DESCRIPTION
**Pull request by tpmcgowan**

### CODE REVIEW

File: server/utils/nunjucksSetup.ts [hunk 1]
```diff
@@ -24,9 +23,13 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
...
   }
 
+  registerNunjucks(app)
+}
+
+export function registerNunjucks(app?: express.Express): Environment {
   const njkEnv = nunjucks.configure(
     [
-      path.join(__dirname, '../../server/views'),
+      'server/views',
```
Comment (tpmcgowan) [inline server/utils/nunjucksSetup.ts:32]: Had to juggle template paths around a fair bit to get thing working when both running normally and via tests.

File: server/routes/visitors.test.ts [hunk 2]
```diff
@@ -201,33 +201,22 @@ describe('POST /visit/select-visitors/A1234BC', () => {
         expect(res.text).toContain('28 July 1986<br>(Adult)')
         expect(res.text).toContain('Sister')
         expect(res.text).toContain('123 The Street')
-        expect(res.text).toContain('visitor-restriction-badge--BAN">Banned</span> until 31 July 2022')
```
Comment (tpmcgowan) [inline server/routes/visitors.test.ts:204]: These assertions were incorrectly passing. Fixed by explicitly handling `err` and calling `done()` in `end()`. Took me a while to figure out what was going on here!

File: package.json [hunk 3]
```diff
@@ -22,11 +22,11 @@
     "lint": "eslint . --cache --max-warnings 0",
     "lint-fix": "eslint . --fix",
     "typecheck": "tsc && tsc -p integration_tests",
-    "test": "jest -i",
+    "test": "jest -i --coverage",
```
Comment (tpmcgowan) [inline package.json:25]: Turned coverage report on for `npm test` - though could add it as a separate task if you prefer....?

Comment (psoleckimoj) [reply package.json:25]: Fine with me here 👍

**Review by psoleckimoj — DISMISSED** [review]
LGTM!

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**